tequilapi/endpoints/health_check.go

```diff
@@ -3,28 +3,35 @@ package endpoints
 import (
 	"github.com/julienschmidt/httprouter"
 	"github.com/mysterium/node/tequilapi/utils"
+	"github.com/mysterium/node/version"
 	"net/http"
 	"time"
 )
 
 type healthCheckData struct {
-	Uptime  string `json:"uptime"`
-	Process int    `json:"process"`
-	Version string `json:"version"`
+	Uptime  string    `json:"uptime"`
+	Process int       `json:"process"`
+	Version buildInfo `json:"version"`
+}
+
+type buildInfo struct {
+	Commit      string `json:"commit"`
+	Branch      string `json:"branch"`
+	BuildNumber string `json:"buildNumber"`
```

Comment:
`buildInfo` is identical to `version.Info`, except `json` names. Is this intentional to make this module less coupled to `version` package..?

Reply:
yes